pkg/virt-handler/vm.go

```diff
@@ -2683,6 +2722,18 @@ func (d *VirtualMachineController) vmUpdateHelperMigrationTarget(origVMI *v1.Vir
 		return nil
 	}
 
+	// Verify container disks checksum
+	err = d.containerDiskMounter.VerifyChecksums(vmi)
```

Comment:
If I understand correctly, this will change the existing behavior. In the case when floating tags are used for containerdisk images, with this change the migration will fail if a tag points to a newer version. The current implementation is sort of agnostic to that, since it refers to the image by its digest. Probably the topic 'whether the usage of floating tags is a good practice or not' is debatable, but I think that if we decide to go with this approach, it should be clearly documented that floating tags will likely cause migration failure with crio >= 1.28. WDYT?

Reply:
Unfortunately yes, we will break some users because Kubernetes doesn't require to report digest. I believe this is the only reasonable handling. Documenting this and discouraging users from using floating tags is a must. Note: This will be backported to 1.0 and 1.1